corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -200,18 +200,18 @@ FormplayerFrontend.on("start", function (options) {
         }
     }
 
-    if (options.allowedOrigin) {
+    if (options.allowedHost) {
         window.addEventListener(
             "message",
-            new FormplayerFrontend.HQ.Events.Receiver(options.allowedOrigin),
+            new FormplayerFrontend.HQ.Events.Receiver(options.allowedHost),
             false
         );
     }
 });
 
 FormplayerFrontend.on('navigation:back', function() {
     var url = Backbone.history.getFragment();
-    if (!url.startsWith('/single_app')) {
+    if (url.indexOf('single_app') === -1) {
```

Comment:
could use `includes` to be slightly clearer? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes